src/main/java/io/stargate/sgv2/jsonapi/StargateJsonApi.java

```diff
@@ -416,6 +416,63 @@
                         }
                       }
                       """),
+              @ExampleObject(
+                  name = "createTableCompositePrimaryKey",
```

Comment:
lets discuss the implications of including this, the code is only alpha and it's going to change. If we go down this path how many commands would we need to keep up to date ?

Reply:
talking - ok to remove and we will fix swagger when we make things beta (or decide to drop swagger specs)